macros/src/lib.rs

```diff
@@ -513,16 +513,22 @@ fn log(level: Level, log: FormatArgs) -> TokenStream2 {
     let sym = mksym(&ls, level.as_str(), true);
     let logging_enabled = cfg_if_logging_enabled(level);
     quote!({
-        #[#logging_enabled] {
-            match (#(&(#args)),*) {
-                (#(#pats),*) => {
-                    if let Some(mut _fmt_) = defmt::export::acquire() {
-                        _fmt_.header(&defmt::export::istr(#sym));
-                        #(#exprs;)*
-                        _fmt_.finalize();
-                        defmt::export::release(_fmt_)
+        cfg_if::cfg_if! {
```

Comment:
This breaks if the end-user's crate doesn't have cfg_if as a dependency

Reply:
Should't it be okay, since I've added it to `defmt`s dependencies?

Reply:
Or do you mean if someone uses `defmt-macros` directly?

Reply:
I see 🤔